src/common/src/session_config/search_path.rs

```diff
@@ -50,30 +52,25 @@ impl SearchPath {
 impl Default for SearchPath {
     fn default() -> Self {
         [USER_NAME_WILD_CARD, DEFAULT_SCHEMA_NAME]
-            .as_slice()
-            .try_into()
+            .join(SESSION_CONFIG_LIST_SEP)
+            .parse()
             .unwrap()
     }
 }
 
-impl ConfigEntry for SearchPath {
-    fn entry_name() -> &'static str {
-        CONFIG_KEYS[SEARCH_PATH]
-    }
-}
-
-impl TryFrom<&[&str]> for SearchPath {
-    type Error = RwError;
+impl FromStr for SearchPath {
+    type Err = RwError;
 
-    fn try_from(value: &[&str]) -> Result<Self, Self::Error> {
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let paths = s.split(SESSION_CONFIG_LIST_SEP).map(|path| path.trim());
```

Comment:
As `SESSION_CONFIG_LIST_SEP = ", "`, so the trailing space is required for the search_path (as well as any other list-like variables)?

Reply:
Tested on PG:

```
test=# SET search_path TO myschema,public;
SET
test=# SET search_path TO myschema, public;
SET
test=# SET search_path TO myschema  , public;
SET
```

Reply:
PG and this PR do the same:
* Parser parses these as a list of values, separated by Token::Comma and ignoring whitespaces.
* Handler joins the list of values to a string separated by `, ` (`flatten_set_variable_args` in `guc_funcs.c`)
* When consuming, split the string into a list of values.